template.yaml

```diff
@@ -70,7 +70,6 @@ Resources:
             Method: get
             RequestParameters:
               - method.request.querystring.language
```

Comment:
For the /unit/{id} path we are missing the pathparam in the RequestParameters list